definitions/types/azure-storage-account-endpoint.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "title": "Azure Storage Account Authentication",
+  "description": "Azure Storage Account endpoint authentication",
+  "additionalProperties": false,
+  "examples": [
+    "https://storageaccount.blob.core.windows.net/",
+    "http://storageaccount.file.core.windows.net/",
+    "abfs://filesystem.accountname.dfs.core.windows.net/"
+  ],
+  "oneOf": [
+    {
+      "type": "string",
+      "pattern": "((http|https):\/\/)[a-zA-Z0-9]+\\.blob\\.core\\.windows\\.net\/$",
+      "message": {
+        "pattern": "Must be a valid Azure Storage Account blob endpoint"
+      }
+    },
+    {
+      "type": "string",
+      "pattern": "^((http|https):\/\/)[a-zA-Z0-9]+\\.queue\\.core\\.windows\\.net\/$",
+      "message": {
+        "pattern": "Must be a valid Azure Storage Account queue endpoint"
+      }
+    },
+    {
+      "type": "string",
+      "pattern": "^((http|https):\/\/)[a-zA-Z0-9]+\\.table\\.core\\.windows\\.net\/$",
+      "message": {
+        "pattern": "Must be a valid Azure Storage Account table endpoint"
```

Comment:
Does this bundle produce these Artifacts or does it require a container or Queued attached before you have an endpoint

Reply:
The endpoints are already available for each storage account created. If you needed a table, you would just use the `table` endpoint for the REST API to take actions within the storage account. Same if a container didn't exist, but you wanted to create one. The endpoint to create a container would look like this: `https://myaccount.blob.core.windows.net/mycontainer?restype=container`

Reply:
Ok, so a dev would use that URL to make all the containers there app wants, kind of how we were discussing it hopefully working when we were considering putting containers here. 

If so _thats great_